Makefile

```diff
@@ -12,6 +12,19 @@ deps vendor_deps: check_setup
 	npm prune
 	echo "DONE.  Consider running:  git add -A vendor node_modules"
 
+# bump the version for upgrade
+version = v1.19.1
```

Comment:
I don't think you need this line anymore.

Reply:
you are right. we pin the verson in go.mod. no-longer need it here. Remove it.